jep/7/README.adoc

```diff
@@ -92,6 +140,7 @@ There are currently two Git repositories holding different states of its source
 It is unmaintained and the changelog mentions a version 0.13 that has never actually been released.
 
 Over the past year, multiple changes to core negatively impacted ruby-runtime based plugins, and core maintainers had to implement workarounds to address these problems.
```

Comment:
Given this JEP is a few years old, phrases like _the past year_ need updating.

Reply:
```suggestion
Over the past years, multiple changes to core negatively impacted ruby-runtime based plugins, and core maintainers had to implement workarounds to address these problems.
```

Will review the whole doc later, just a minor bump